<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Juego INFO198</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number> </property>
      <widget class="QWidget" name="ConnectionPage">
       <layout class="QVBoxLayout" name="verticalLayout_Conn">
        <item>
         <spacer name="verticalSpacer_ConnTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QWidget" name="widget_ConnForm" native="true">
          <layout class="QFormLayout" name="formLayout_Conn">
           <item row="0" column="0">
            <widget class="QLabel" name="label_IP">
             <property name="text">
              <string>Dirección IP Servidor:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ipInput">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="placeholderText">
              <string>Ej: 192.168.1.10</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_Name">
             <property name="text">
              <string>Nombre de usuario:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nameInput">
             <property name="text">
              <string>Jugador</string>
             </property>
             <property name="placeholderText">
              <string>Tu nombre</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_ConnButtons">
          <item>
           <spacer name="horizontalSpacer_ConnLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="hostButton">
            <property name="text">
             <string>Crear partida</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="joinButton">
            <property name="text">
             <string>Unirse a partida</string>
            </property>
           </widget>
          </item>
          <item>
            <widget class="QPushButton" name="exitButton_ConnectionPage">
             <property name="text">
              <string>Salir</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #6c757d; color: white;</string>
             </property>
            </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_ConnRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_ConnBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="LobbyPage">
       <layout class="QVBoxLayout" name="verticalLayout_Lobby">
        <item>
         <widget class="QLabel" name="label_TeamName">
          <property name="text">
           <string>Ingresa el nombre de tu equipo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="teamNameInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="addPlayerButton">
          <property name="text">
           <string>Unirse/Crear Equipo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_LobbyStatus">
          <property name="text">
           <string>Estado de la Sala:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="lobbyDisplay">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="startGameButton">
          <property name="text">
           <string>Empezar Juego (Anfitrión)</string>
          </property>
          <property name="enabled">
           <bool>false</bool> </property>
         </widget>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="GamePage">
       <layout class="QHBoxLayout" name="horizontalLayout_Game">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_GameLog">
          <item>
           <widget class="QLabel" name="label_GameLog">
            <property name="text">
             <string>Registro del Juego:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="logDisplay">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximumWidth">
             <number>300</number> </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_GameArea">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Turn">
            <item>
             <widget class="QPushButton" name="nextTurnButton">
              <property name="text">
               <string>Lanzar Dado</string>
              </property>
              <property name="enabled">
               <bool>false</bool> </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="turnStatusLabel">
              <property name="text">
               <string>Esperando turno...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_Turn">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="gameAreaWidget"> <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>100</width> <height>100</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="gameLayout">
              <item>
               <widget class="QLabel" name="label_PlaceholderGame">
                <property name="text">
                 <string>Área del Juego (Puntuaciones, etc.)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_GameArea">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="WinnerPage">
       <layout class="QVBoxLayout" name="verticalLayout_Winner">
        <item>
         <spacer name="verticalSpacer_WinTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="winnerLabel">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>¡¡Equipo X ha ganado!!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_WinMid">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="lobbyButton">
          <property name="text">
           <string>Volver a la Sala</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_WinBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
